packages/contracts/script/DefaultSetupScript.s.sol

```diff
@@ -0,0 +1,217 @@
+            unclaimsHandler = UnclaimsHandler(payable(address(proxy)));
+        }
+
+        oracle = new UniswapTWAPOracle(uniswapFactory, weth);
+
+        testToken = new TestERC20("EmailWalletV1Test", "EWV1TEST", maxAmount);
+
+        {
+            coreImpl = new EmailWalletCore();
+            bytes memory data = abi.encodeWithSelector(
+                EmailWalletCore(coreImpl).initialize.selector,
+                    address(relayerHandler),
+                    address(accountHandler),
+                    address(unclaimsHandler),
+                    address(extensionHandler),
+                    address(verifierImpl),
+                    address(tokenRegistry),
+                    address(oracle),
+                    address(weth),
+                    2 gwei,
+                    1 hours,
+                    450000,
+                    500000
+            );            
+            ERC1967Proxy proxy = new ERC1967Proxy(address(coreImpl), data);
+            core = EmailWalletCore(payable(address(proxy)));
+            core.relayerHandler().transferOwnership(address(core));
+            core.accountHandler().transferOwnership(address(core));
+            core.unclaimsHandler().transferOwnership(address(core));
+            core.extensionHandler().transferOwnership(address(core));
+        }
+
+        vm.stopBroadcast();
+
+        console.log("TokenRegistry proxy deployed at: %s", address(tokenRegistry));
+        console.log("TokenRegistry implementation deployed at: %s", address(tokenRegistryImpl));
+        console.log("AllVerifiers implementation deployed at: %s", address(verifierImpl));
+        console.log("ECDSAOwnedDKIMRegistry deployed at: %s", address(dkim));
+        console.log("Wallet implementation deployed at: %s", address(walletImpl));
+        console.log("RelayerHandler proxy deployed at: %s", address(relayerHandler));
+        console.log("RelayerHandler implementation deployed at: %s", address(relayerHandlerImpl));
+        console.log("ExtensionHandler proxy deployed at: %s", address(extensionHandler));
+        console.log("ExtensionHandler implementation deployed at: %s", address(extensionHandlerImpl));
+        console.log("AccountHandler proxy deployed at: %s", address(accountHandler));
+        console.log("AccountHandler implementation deployed at: %s", address(accountHandlerImpl));
+        console.log("UnclaimsHandler proxy deployed at: %s", address(unclaimsHandler));
+        console.log("UnclaimsHandler implementation deployed at: %s", address(unclaimsHandlerImpl));
+        console.log("UniswapTWAPOracle deployed at: %s", address(oracle));
+        console.log("TestERC20 deployed at: %s", address(testToken));
+        console.log("EmailWalletCore proxy deployed at: %s", address(core));
+        console.log("EmailWalletCore implementation deployed at: %s", address(coreImpl));  
+        console.log("---- DONE ----");
```

Comment:
This script contains 01 - 06 scripts, right?
Is the 07 script not needed for this?

Reply:
I wanted to keep it as a default script. I can add extensions too if required.